apps/fishing-map/features/map/error-notification/ErrorNotification.tsx

```diff
@@ -0,0 +1,117 @@
+import { useSelector } from 'react-redux'
+import { Popup } from 'react-map-gl'
+import { useTranslation } from 'react-i18next'
+import { KeyboardEventHandler, useState } from 'react'
+import { Button, Icon, InputText } from '@globalfishingwatch/ui-components'
+import { GUEST_USER_TYPE } from '@globalfishingwatch/api-client'
+import { useMapErrorNotification } from 'features/map/error-notification/error-notification.hooks'
+import { loadSpreadsheetDoc } from 'utils/spreadsheet'
+import { selectUserData } from 'features/user/user.slice'
+import { EMPTY_FIELD_PLACEHOLDER } from 'utils/info'
+import { selectErrorNotification } from './error-notification.slice'
+import styles from './ErrorNotification.module.css'
+
+const ERRORS_SPREADSHEET_ID = '1VzEjTiSJrbwBfZrhaSFwZgHClxKkwN9E0rrfFR8bw2c'
+const ERRORS_SHEET_TITLE = 'errors'
+
+const ErrorNotification = () => {
+  const { t } = useTranslation()
+  const errorNotification = useSelector(selectErrorNotification)
+  const { resetErrorNotification, setErrorNotification, setNotifyingError } =
+    useMapErrorNotification()
+  const [loading, setLoading] = useState(false)
+  const [success, setSuccess] = useState(false)
+  const userData = useSelector(selectUserData)
+
+  if (!errorNotification) {
+    return null
+  }
+
+  const onConfirmClick = async () => {
+    setLoading(true)
+    try {
+      const feedbackSpreadsheetDoc = await loadSpreadsheetDoc(ERRORS_SPREADSHEET_ID)
+      // loads document properties and worksheets
+      const sheet = feedbackSpreadsheetDoc.sheetsByTitle[ERRORS_SHEET_TITLE]
+      const date = new Date()
+      const annotationURL = `&mapAnnotations[0][lon]=${
```

Comment:
I'd try to stringify it using the same library we have for the URL instead of hardcoding it
https://github.com/GlobalFishingWatch/frontend/blob/master/libs/dataviews-client/src/url-workspace/url-workspace.ts#L259